Dockerfile

```diff
@@ -47,10 +30,12 @@ ENV NODE_ENV=production
 RUN apk del .build-dependencies
 # Copy only `node_modules` needed to run the server
 COPY --from=dependencies /app/production_node_modules ./node_modules
-# @godaddy/terminus has an example folder with a package.json file 
+# @godaddy/terminus has an example folder with a package.json file
```

Comment:
do these comments still have their place here?

Reply:
I don't think so, but it was there, I left it in place

Reply:
OK, I think they are OK. I understood why they are here by looking at the complete file. We remove files from the image to not have aquascanner fail. So the comments are relevant